source/plugin/data/index.rst

```diff
@@ -0,0 +1,78 @@
+============
+The Data API
+============
+
+Overview
+========
+
+The unified Data API aims to provide a consistent way of accessing and modifying data of any kind. Where other
+approaches define the available data using interface and inheritance (like a ``LivingEntity`` interface providing
+getter and setter functions for current and maximum health), in Sponge every entity, block etc. is completely
+oblivious to what data it holds. While this may appear less straightforward than direct accessor methods, it is
+foremost far more extensible. And thanks to the addition of ``Key``\ s, simply accessing specific values is no
+less straightforward.
+
+.. warning::
+
+    As of writing, the Data API is not yet fully implemented. Refer to the `Implementation Tracker
+    <https://github.com/SpongePowered/SpongeCommon/issues/8>`_, ask in the ``#spongedev`` IRC channel or on the
+    `Forums <https://forums.spongepowered.org>`_ to find out if the data you need to work with is available yet.
+
+Concepts
+========
+
+On first glance at the API docs, the data API threatens to overwhelm you with lots of interfaces and packages. But
+to simply use the data API, you will not have to deal with many of them, as most interfaces found there are just
+specific data manipulators.
+
```

Comment:
If they are the backend, should they really get included into the basic section? I tried to include only those which a plugin developer will directly interact with. Are `ValueContainer`s relevant for basic users or should they rather be spared until documentation for advanced users follows?